hydra/test_utils/config_source_common_tests.py

```diff
@@ -70,7 +94,6 @@ def test_is_config(
             ("level1", ObjectType.GROUP, ["level2"]),
             ("level1/level2", ObjectType.GROUP, []),
             # Configs
-            ("", ObjectType.CONFIG, ["config_without_group", "dataset"]),
```

Comment:
Unless there is an additional problem I am not aware of, `config_without_group` should not be skipped.